app/transitions/reverse-work-explosion.js

```diff
@@ -0,0 +1,17 @@
+import Ember from 'ember';
+import { Promise } from "liquid-fire";
+import { TimelineLite, easing } from 'gsap';
+
+export default function(opts = {}) {
+  const ease = easing.Expo.easeInOut;
+  const { newElement, oldElement } = this;
+  let requiredX = oldElement.offset().left - newElement.offset().left;
+  return new Promise(resolve => {
+    const tl = new TimelineLite({ onComplete: resolve });
+    tl.to(Ember.$('.detectScroll').get(0), 0.3, { scrollTo: newElement.get(0), ease });
+    if (requiredX) {
+      tl.to(oldElement.get(0), 0.55, { x: -requiredX, ease: easing.Expo.easeOut });
```

Comment:
does the requiredX 'scroll' the container back to where it started?

Reply:
yup, on the X axis ( let requiredX = oldElement.offset().left - newElement.offset().left; )

This may change a lil in a future PR tho